src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -90,10 +93,14 @@ func Configure(addr string, cloudConfigCA string, instanceId string,
 
 	// Using a goroutine because we'll be using waitforserver and at this time
 	// the proxy is not yet ready.
+	done = make(chan bool, 1)
```

Comment:
I did this in order to provide optional blocking on the library client side. It doesn't change the API so it won't affect existing code.